libvirt/resource_libvirt_network.go

```diff
@@ -92,6 +92,12 @@ func resourceLibvirtNetwork() *schema.Resource {
 				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"enable": {
```

Comment:
According to libvirt/resource_libvirt_network.go:212, DHCP uses "enabled" with a "d". I think it would be a  good idea to use same syntax.

Reply:
@Bischoff I was following the same syntax which https://libvirt.org/formatnetwork.html#elementsAddress (dns section) and also the structure defined in the `libvirt-xml` https://github.com/libvirt/libvirt-go-xml/blob/master/network.go#L220 , I was hoping that will be easy for user coming from libvirt side, since for `dhcp` there is no convention as per document. 

But if you folks think that would be make more sense, I am happy to make it :)

Reply:
I think enabled would be more coherent along this codebase. Thx

Reply:
You're referring to libvirt syntax. Currently terraform libvirt provider syntax differs from it.

We could stick to libvirt syntax, but then we would need to change it for DHCP, for consistency sake.
@dmacvicar any thoughts?

Reply:
Is better to change this then changing an existing API. A part of that enabled looks to me more logic the enable.

Reply:
@Bischoff @MalloZup updated to `enabled` now.